src/content/docs/integrations/ci-further.mdx

```diff
@@ -1,3 +1,7 @@
+---
+title: 'Ci further'
+---
```

Comment:
Is this mandatory? Should we prefix the file with `_` to avoid this?
(the page is only used for include)

Reply:
Done